grpc/src/client.rs

```diff
@@ -91,7 +91,10 @@ impl CommunicationsClient for GRPCCommunicationsClient {
                         // [impl->swdd~grpc-client-outputs-error-server-unavailability-for-cli-connection~1]
                         Err(GrpcMiddlewareError::ServerNotAvailable(err)) => {
                             log::debug!("No connection to the server: '{err}'");
-                            return Err(CommunicationMiddlewareError("No connection to the server! Make sure that Ankaios Server is running.".to_string()));
+                            return Err(CommunicationMiddlewareError(format!(
+                                "Could not connect to Ankaios server on {}.",
```

Comment:
```suggestion
                                "Could not connect to Ankaios server on '{}'.",
```

Reply:
Hm, so currently the error message is:

```
error: Communication middleware failure: 'Could not connect to Ankaios server on http://127.0.0.1:25551/.'
```

With your suggested change it would be:

```
error: Communication middleware failure: 'Could not connect to Ankaios server on 'http://127.0.0.1:25551/'.'
```

That looks strange to me.

Reply:
The detailed message is already in quotes.